interventions/models.py

```diff
@@ -37,7 +37,7 @@ class Patient(models.Model):
     start_date = models.DateField('Data przyjęcia')
     main_dr = models.ForeignKey(Doctor, on_delete=models.DO_NOTHING, verbose_name = "Lekarz prowadzący")
     dr_description = models.TextField('Opis')
-
+    signed_out_date = models.DateField(None)
```

Comment:
Po pierwsze: ze zmianą modeli powinna się wiązać generacja migracji, bo jeżeli potem ktoś te zmiany pobierze to będzie mu wywalać error i będzie musiał kombinować zanim dojdzie do wniosku, że to po prostu brak migracji. 
Po drugie: z tego co pamiętam to uzgodniliśmy, że będzie bez signed_out_date więc tego sie trzymajmy. Usuń proszę tę modyfikację, dodamy ją w przyszłości jeżeli będzie potrzebna. Na razie dla uproszczenia usuwamy pacjenta całkowicie z bazy.